### DISCUSSION TITLE
Disable Certificate validation for https component

### DISCUSSION THREAD

**Discussion by samar-elsayed** [apache/camel-karavan#1283] [Q&A]
### Ask a question

How can I disable Certificate validation for https component ?
@davsclaus 

### Variant

Vs Code Extension

### Container Management (if applicable)

Kubernetes

### Operating System (if applicable)

Windows

### Version

4.5.0

**Comment by davsclaus** [comment]
you are welcome to share some details on the solution in case others see the discussion and want to know the answer or get help
